Engine/Login/LoginCliAction.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using OpenTap.Cli;
+
+namespace OpenTap.Login
+{
+    /// <summary>
+    /// Logs in to an Keycloak instance.
```

Comment:
Is this only meant to be used against Keycloak? If `yes` shouldn't we highlight it in the name?